configs/components/fesom/Subjob_tripy.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+#Home directory and reset of variable
+cd ~
```

Comment:
The script should be runnable from anywhere, so I would recommend that you don't switch directories too much, especially not outside of the current experiment structure unless you have a good reason to.